src/app/docs.js

```diff
@@ -1 +1,6 @@
-import TextDoc from './components/Text/Text.docs';const DOCS = {}; DOCS["Text"] = TextDoc; export default DOCS;
\ No newline at end of file
+import TextDocs from './components/Text/Text.docs';
+
+const DOCS = {}; 
+DOCS["Text"] = TextDocs; 
+
+export default DOCS;
```

Comment:
I reformatted it while troubleshooting why the lookup was failing to import the docs. The problem was in the component pulling them in, not the autogen